streams/src/main/java/org/apache/kafka/streams/processor/internals/StoreChangelogReader.java

```diff
@@ -1012,6 +1017,8 @@ private void prepareChangelogs(final Map<TaskId, Task> tasks,
                 // no records to restore; in this case we just initialize the sensor to zero
                 final long recordsToRestore = Math.max(changelogMetadata.restoreEndOffset - startOffset, 0L);
                 task.recordRestoration(time, recordsToRestore, true);
+            }  else if (changelogMetadata.stateManager.taskType() == TaskType.STANDBY) {
+                standbyUpdateListener.onUpdateStart(partition, storeName, startOffset);
```

Comment:
Same here, we should wrap in try-catch that mirrors the #onRestoreStart block above:
```suggestion
                try {
                    standbyUpdateListener.onUpdateStart(partition, storeName, startOffset);
                } catch (final Exception e) {
                    throw new StreamsException("Standby update listener failed on update start", e);
                }
```